script/fetch-issues.js

```diff
@@ -18,13 +18,28 @@ const GH_GQL_OPTIONS = {
 
 const client = GraphQL(GH_GQL_OPTIONS);
 
-// number of pages we want to query (default: 20 pages x 100 items)
+// number of pages we want to query
 // if PAGE_THRESHOLD is -1, then fetch all issues/PRs
-const pageThreshold = process.env.PAGE_THRESHOLD || 20;
+const pageThreshold = process.env.PAGE_THRESHOLD || -1;
 console.log('page number threshold:', pageThreshold);
 
+// earliest date we want to query from
+// GitHub supports reactions since 2016-03-10
+const earliestDate = process.env.EARLIEST_DATE || '2016-03-10T00:00:00Z';
+
 const repo = process.env.REPOSITORIES;
 
+// review will be ignored if it's written by any author specified here
+// delimiter: space
+const ignore_review_author = process.env.IGNORE_REVIEW_AUTHOR ||
```

Comment:
I think the whole project uses camelCase instead of snake_case, need to keep it consistent here.

env is okay though.